### ISSUE TITLE
Generate skybox based on real star data

### ISSUE THREAD

**Issue by aggregate1166877** [feynmansbongos/Cosmosis#59]
Details:
While implementing stars, it was found that 1) depth testing in the graphics card causes serious glitches for distant overlapping stars, but is easily fixed by disabling depth testing and 2) alpha blending significantly improves how stars look in general.

The issue however is that these ideas do not work with 3D models such as space ships. We really do want the best of both worlds though, so we need to render the stars and the 3D world separately. 

~~A design decision needs to be made as part of this task: render all stars each frame, or render stars only after significant distance change (eg. 1AU)?~~ - this will be done as a separate task.

Current tests show that rendering 9110 stars from the BSC5P on integrated graphics (Vega 10) renders at 120FPS. It may nonetheless help to render the stars as a skybox as intended travel in this game is star hopping, not travelling at millions of times the speed of light. The game may therefore benefit from not rerendering everything each frame. Something to keep in mind is that we'll still need to render thousands of stars each frame in the galaxy overview, but this will be implemented as a different scene where z-order might not matter anyway.

~~Something like this might be of use if going the skybox route: https://github.com/spite/THREE.CubemapToEquirectangular~~
 ^^-->> turned out to be unnecessary. I just take the the image blob and manually set all 6 faces of the skybox.

~~Nearby stars (<5ly) would then not be generated as part of the skybox, but rather as part of points in the 3D world.~~ - this will be done as part of another task.

**Comment by aggregate1166877** [comment]
This is done. Fun problem to solve.
